CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE code repo](README.md), check out the [Contributor's Guide](https://github.com/o3de/community/blob/main/CONTRIBUTING.md).
```

Comment:
This link is pointed to the main branch, but the PR is going into development?

Reply:
I believe the community repository only has a main branch